dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventChangeLogService.java

```diff
@@ -128,6 +134,23 @@ public void addDataValueChangeLog(
     hibernateEventChangeLogStore.addEventChangeLog(eventChangeLog);
   }
 
+  @Override
+  @Transactional
+  public void addPropertyChangeLog(Event newEvent, Event currentEvent, String userName) {
```

Comment:
Keep the order and names consistent with the other method

You have `addDataValueChangeLog`

      String previousValue,
      String currentValue,

vs `addPropertyChangeLog`

    Event newEvent
    Event currentEvent

You might even give `current` two meanings 🤔 

previousValue/Event and currentValue/Event make sense to me and the order as well.